database.cc

```diff
@@ -884,13 +884,25 @@ void database::add_column_family(keyspace& ks, schema_ptr schema, column_family:
     schema = local_schema_registry().learn(schema);
     schema->registry_entry()->mark_synced();
 
+    auto needs_repair_before_gc = [this, ks_name = schema->ks_name()] {
+        // If a table uses local replication strategy or rf one, there is no
+        // need to run repair even if tombstone_gc mode = repair.
+        auto& ks = find_keyspace(ks_name);
```

Comment:
It's sad that you have to perform this lookup. We should switch database::_keyspaces to contain unique_ptr<keyspace> so keyspaces can't move around.